src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -24,6 +24,7 @@
     private static final String[] ACTIONS = new String[]{
             "internalStart", "internalDock", "internalAuction", "internalGalaxyGates", "internalPilotSheet"};
 
+    public final HangaManager hangaManager;
```

Comment:
Is it not possible to keep the API (the public methods) of the older HangarManager in the new one, instead of making this bootleag `HangaManager`?